runtime/shared/shrclssup.c

```diff
@@ -276,6 +276,8 @@ IDATA J9VMDllMain(J9JavaVM* vm, IDATA stage, void* reserved)
 			}
 			vm->sharedCacheAPI->runtimeFlags = runtimeFlags;
 		}
+
+		vm->sharedCacheAPI->xShareClassCacheDisabledOnCRIURestore = FALSE;
```

Comment:
Do you need to set `xShareClassCacheDisabledOnCRIURestore` to `FALSE` ? Line 53 memset the whole `sharedCacheAPI` struct to 0 already.

Reply:
removed in [86ce34e](https://github.com/eclipse-openj9/openj9/commit/86ce34e9e82cb6489368e41b35512d4681859702)